sdk/python/kfp/client/client_test.py

```diff
@@ -41,5 +47,89 @@ def test_invalid(self, name: str):
             client.validate_pipeline_resource_name(name)
 
 
+PIPELINES_TEST_DATA_DIR = os.path.join(
+    os.path.abspath(os.path.join(__file__, os.path.pardir, os.path.pardir)),
+    'compiler', 'test_data', 'pipelines')
```

Comment:
I would avoid referencing test samples across packages. It would be strange if some one changed and test sample for compiler but found it broke client UT. I think what you have below w/o dependencies on compiler test data is already sufficient.